static/extensions/Faunks/Blobs.js

```diff
@@ -103,10 +128,43 @@
     }
 
     toBlob(args, util) {
-        const text = String(args.DATA);      // ensure it's a string (why can't it be just like python or smth)
+        const text = String(args.DATA); 
         const blob = new Blob([text], { type: args.TYPE });
         return URL.createObjectURL(blob);
-    } 
+    }
+    
+    revokeBlob(args, util){
+        URL.revokeObjectURL(args.URL);
+    }
+
+    readBlob(args, util) {
+    async function readBlobContent(url, mime) {
+        
+        if (!url.startsWith("blob:")) return ""; // I WOULD return "Url Error", but it would make it a bit harder to track if an output has been returned or not
```

Comment:
You could throw an error. So ``throw new URIError("must be a blob url");`` or something similar.